packages/ts/react-crud/test/autoform.spec.tsx

```diff
@@ -43,6 +67,14 @@ describe('@hilla/react-crud', () => {
         .map(([, value]) => value.toString());
     }
 
+    function TestFieldRenderer({ field: Field, propertyInfo, ...other }: AutoFormFieldRendererProps) {
+      return <Field data-testid={propertyInfo.name} propertyInfo={propertyInfo} {...other} />;
```

Comment:
Why do we need test ids? Every field should have a label that we can query for. That would also implicitly test that labels are set correctly, which is missing currently.

Reply:
It also seems wrong to add this field renderer abstraction to the public API of the component, just so that we can test it.

Reply:
I removed test ids from the package. They are also used in `react-form`, but that's outside of the scope of this PR.